app/controllers/api_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ApiController < ApplicationController
+  def service_interaction_for_local_authority
+    @authority = LocalAuthority.find_by_slug!(params[:authority_slug])
+    @service = Service.find_by(lgsl_code: params[:lgsl])
+    interactions = @service.interactions
+    @link = links.detect { |l| interactions.first.service_interactions.first.id == l.service_interaction.id }
+
+    render json: local_interaction_response
+  end
+
+private
+
+  def local_interaction_response
+    {
+      "local_authority"=>{
+        "name"=>@authority.name,
```

Comment:
Just mentioned [this](https://github.com/alphagov/frontend/pull/977#discussion_r72037802) in the Frontend PR. Maybe we shouldn't be combining the two types of data (local authority data/local interaction data) in a single route/request. Instead could have 2 separate routes, one for local authority data and one for local interaction data. Frontend would have to make 2 requests to get the appropriate data.